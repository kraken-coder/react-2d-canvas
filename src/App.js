import React from 'react'
import './App.css'

const App = () => (
  <div className="app">
    <div className="section">
      <h1 className="heading">React App Skelaton</h1>
      {/* <div className="inner">
        <p className="small-text">A very powerful alternative to create react app</p>
      </div> */}
    </div>
  </div>
)

export default App
