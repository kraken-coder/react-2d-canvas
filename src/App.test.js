import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'
import React from 'react'
import App from './App'

test('match snapshot', () => {
  const Wrapper = shallow(<App />)
  expect(toJSON(Wrapper)).toMatchSnapshot()
})
