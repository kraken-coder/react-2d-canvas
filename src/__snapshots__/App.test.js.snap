// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`match snapshot 1`] = `
<div
  className="app"
>
  <div
    className="section"
  >
    <h1
      className="heading"
    >
      React App Skelaton
    </h1>
  </div>
</div>
`;
